hub-config/admin.json

```diff
@@ -13,5 +13,13 @@
     },
     "file_format": ["csv"],
     "timezone": "US/Eastern",
-    "model_output_dir": "model-output"
+    "model_output_dir": "model-output",
+    "cloud": {
+        "enabled": false,
+        "host": {
+          "name": "aws",
+          "storage_service": "s3",
+          "storage_location": "hubverse-complex-forecast-example"
```

Comment:
I thought it seemed more tidy to have the same prefix for all of our Hubverse-maintained S3 buckets, but happy to update if people prefer to have the name of the S3 bucket match the repo name.